README.rst

```diff
@@ -102,3 +102,51 @@ Perform a query
 
     for row in rows:
         print(row.name)
+
+Instrumenting With OpenTelemetry
+--------------------------------
+
+This application uses OpenTelmetry to output tracing data from
+api calls to BigQuery. To enable OpenTelemetry tracing in
+the BigQuery client the following PyPi packages need to be installed:
+
+.. code-block:: console
+
+    pip install opentelemetry-api
+
+.. code-block:: console
+
+    pip install opentelemetry-sdk
+
+.. code-block:: console
+
+    pip install opentelemetry-instrumentaion
+
+.. code-block:: console
+
+    pip install opentelemetry-exporter-cloud_trace
+
+After installation, OpenTelemetry can now be used in the BigQuery
+client and in BigQuery jobs. First however an exporter must be
+specified for where the trace data will be outputted to. An example of this
+can be found here:
+
+.. code-block:: python
+
+    from opentelemetry import trace
```

Comment:
Is there a runnable / tested code snippet somewhere this can be linked from? I'm worried that the code sample may get out-of-date / break.

Reply:
Currently looking into it.